release.sh

```diff
@@ -1,15 +1,14 @@
 # Before you release
-# - install pandoc on your system: apt-get install pandoc
-# - install pypandoc: pip install pypandoc
 # - install twine: pip install twine
-# - bump package version in setup.py
+# - bump package version in setup.cfg
 # - update the package version in the CHANGES file
 # - commit the changes to master and push
 
-PKG_NAME=$(python setup.py --name)
-PKG_VERSION=$(python setup.py --version)
+PKG_NAME = pycobertura
```

Comment:
I dont think we can have spaces around the `=` sign:

```zsh
$ PKG_NAME = pycobertura
zsh: command not found: PKG_NAME
```

But this works:

```zsh
$ PKG_NAME=pycobertura 
$ echo $PKG_NAME 
pycobertura
```

Reply:
yes you are right the shell does not allow whitespaces around = (I am so used to this in the other programming languages that I forget, that the shell does not allow this)